src/Entity/Field.php

```diff
@@ -208,7 +208,11 @@ public function getPhpDocType(bool $allowNullable = true): string
 
         if ($this->isArray() && !$this->isArrayOfObjects()) {
             $arraySuffix = '[]';
-            $typeHint    = $this->getArrayItem()->getPhpTypeHint();
+            $arrayItem = $this->getArrayItem();
+
+            $typeHint = $arrayItem->getType()->isMixed()
+                ? self::TYPE_MIXED
+                : $arrayItem->getPhpTypeHint();
```

Comment:
this check should be inside Field.php IMO, like we're doing here:
```php
    public function getPhpDocType(bool $allowNullable = true): string
    {
        if ($this->type->isMixed()) {
            return self::TYPE_MIXED;
        }

        $nullableSuffix = '';
        $arraySuffix    = '';
...
```

Reply:
I have fixed. There was a weird issue as well what's also fixed.
In case of nullable arrays the api generator created this comment:
```
  * @param string|null[] $items
 ```